src/components/DeleteManifestConfirmationModal/__tests__/DeleteManifestConfirmationModal.test.tsx

```diff
@@ -34,6 +34,30 @@ describe('DeleteConfirmationModal', () => {
     expect(getByLabelText('Are you sure you want to delete this manifest?')).toBeInTheDocument();
   });
 
+  it('renders a success notification when manifest is successfully deleted', async () => {
+    render(
+      <div>
+        <DeleteManifestConfirmationModal {...props} />
+      </div>
+    );
+
+    await new Promise((r) => setTimeout(r, 2000));
+
+    waitFor(() => expect(screen.findByText('Manifest deleted')).toBeInTheDocument());
+  });
+
+  it('renders a error message ', () => {
+    const { findByText } = render(
+      <div>
+        <DeleteManifestConfirmationModal {...props} />
+      </div>
+    );
+
+    waitFor(() =>
+      expect(screen.findByText('Something went wrong. Please try again')).toBeInTheDocument()
+    );
+  });
```

Comment:
Is there a reason we are updating this test if we aren't updating the component?